problems/my_join.rb

```diff
@@ -1,8 +1,8 @@
-# Monkey patch the Array class and add a my_join method. If my_join receives
+# Write an `Array#my_join` method. If my_join receives
 # no argument, then use an empty string to join the array together.
 # ex.
-# [1,2,3].my_join => '123'
-# [1,2,3].my_join('$') => '1$2$3'
+# `[1,2,3].my_join` => '123'
+# `[1,2,3].my_join('$')` => '1$2$3'
```

Comment:
Add note similar to above explicitly stating that they are not allowed to use built-in `Array#join` method.